influxdb_client/client/influxdb_client.py

```diff
@@ -83,14 +83,16 @@ def __exit__(self, exc_type, exc_value, traceback):
         self.close()
 
     @classmethod
-    def from_config_file(cls, config_file: str = "config.ini", debug=None, enable_gzip=False, **kwargs):
+    def from_config_file(cls, config_file: str = "config.ini", debug=None, enable_gzip=False,
+                         config_name: str = "influx2", **kwargs):
         """
         Configure client via configuration file. The configuration has to be under 'influx' section.
 
         :param config_file: Path to configuration file
         :param debug: Enable verbose logging of http requests
         :param enable_gzip: Enable Gzip compression for http requests. Currently, only the "Write" and "Query" endpoints
                             supports the Gzip compression.
+        :param config_name: Name of the configuration section of the configuration file
```

Comment:
Please use `kwargs` instead of new argument:
```suggestion
        :key config_name: Name of the configuration section of the configuration file
```